app/MMB/src/scenes/tripServices/insurance/InsuranceOverview.js

```diff
@@ -0,0 +1,44 @@
+// @flow strict
+
+import * as React from 'react';
+import { LayoutSingleColumn } from '@kiwicom/mobile-shared';
+import { graphql, PrivateApiRenderer } from '@kiwicom/mobile-relay';
+
+import DestinationImage from './DestinationImage';
+import TripInfo from '../../../components/header/TripInfo';
+import BookingDetailContext from '../../../context/BookingDetailContext';
+import type { InsuranceOverviewQueryResponse } from './__generated__/InsuranceOverviewQuery.graphql';
+
+function InsuranceOverview(response: InsuranceOverviewQueryResponse) {
+  return (
+    <LayoutSingleColumn>
+      <DestinationImage data={response.node} />
+      <TripInfo data={response.node} />
```

Comment:
So this is just a dummy view for the time being? It does not seem strictly related to insurance.

Reply:
I am not sure if I understand. This component is reused from the header because this is what we need according to the design.

Reply:
Oh right, sorry.